heracles/__init__.py

```diff
@@ -32,6 +32,27 @@
     "FitsCatalog",
     "FootprintFilter",
     "InvalidValueFilter",
+    # dices
+    "get_cls",
+    "get_delete_cls",
+    "get_bias",
+    "get_delete_fsky",
+    "get_biasjk",
+    "correct_bias",
+    "compute_mask_correction",
+    "correct_mask",
+    "get_delete1_cov",
+    "get_gaussian_target",
+    "get_W",
+    "get_shrinkage",
+    "shrink_cov",
+    "get_delete2_cov",
+    "get_delete2_correction",
+    "get_dices_cov",
+    "Components2Data",
+    "Data2Components",
+    "Components2Fields",
+    "Fields2Components",
```

Comment:
I don't think we want any of these to be available under the `heracles` namespace. I think we want users to do `heracles.dices.get_bias()` etc. because otherwise the names are ambiguous.

Reply:
Addressed

Reply:
addressed!